management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -120,7 +120,7 @@ public static double getExpectedValueFromRatioHistory(String statName, Context c
         int mostRecentIndex = ratioHistory.getValue().length - 1;
         value = ratioHistory.getValue()[mostRecentIndex].getValue();
       }
-    } while (!Thread.currentThread().isInterrupted() && value != expectedResult);
+    } while (!Thread.currentThread().isInterrupted() && !value.equals(expectedResult));
```

Comment:
Instead of having a major single sleep in the test method, I would rather have here a back off strategy of sleeping more and more, from 10ms up to 500ms at a time to break the tight loop.